README.md

```diff
@@ -185,30 +189,105 @@ guarantees.
+| [Jaeger Thrift Exporter](./exporters/jaeger-thrift)         | `opentelemetry-exporter-jaeger-thift`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Jaeger Proto](./exporters/jaeger-proto)                    | `opentelemetry-exporter-jaeger-proto`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Logging Exporter](./exporters/logging)                     | `opentelemetry-exporter-logging`           | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP Logging Exporter](./exporters/logging-otlp)           | `opentelemetry-exporter-logging-otlp`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Zipkin Exporter](./exporters/zipkin)                       | `opentelemetry-exporter-zipkin`            | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Prometheus Exporter](./exporters/prometheus)               | `opentelemetry-exporter-prometheus`        | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+
+### Shim
+
+| Component                              | Artifact ID                      | Version                                                     |
+|----------------------------------------|----------------------------------|-------------------------------------------------------------|
+| [Micrometer Shim](./micrometer1-shim)  | `opentelemetry-micrometer1-shim` | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+| [OpenCensus Shim](./opencensus-shim)   | `opentelemetry-opencensus-shim`  | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+| [OpenTracing Shim](./opentracing-shim) | `opentelemetry-opentracing-shim` | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+
+### Extension
+
+| Component                                                     | Artifact ID                                 | Version                                                     |
+|---------------------------------------------------------------|---------------------------------------------|-------------------------------------------------------------|
+| [Annotations Extension](./extensions/annotations)             | `opentelemetry-extension-annotations`       | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [AWS Extension](./extensions/aws)                             | `opentelemetry-extension-aws`               | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Kotlin Extension](./extensions/kotlin)                       | `opentelemetry-extension-kotlin`            | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Trace Propagators Extension](./extensions/trace-propagators) | `opentelemetry-extension-trace-propagators` | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Incubator Extension](./extensions/incubator)                 | `opentelemetry-extension-incubator`         | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
```

Comment:
It's not really an extension to the incubator, it's incubating APIs that are currently provided as extensions. Maybe

```suggestion
| [Incubating/Experimental new core APIs](./extensions/incubator)                 | `opentelemetry-extension-incubator`         | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
```

Reply:
Changed this header from "Extensions" => "API Extensions" to reflect the difference between these extensions and the SDK extensions.